metricflow/engine/metricflow_engine.py

```diff
@@ -595,12 +595,21 @@ def simple_dimensions_for_metrics(  # noqa: D102
                                     entity_reference.element_name
                                     for entity_reference in linkable_dimension.entity_links
                                 ),
-                                time_granularity=linkable_dimension.time_granularity,
+                                # TODO: Update StructuredLinkableSpecName to use ExpandedTimeGranularity
+                                time_granularity=(
```

Comment:
Should we put a TODO here to make sure we update this to use the str instead of the enum values in the future?

Reply:
Already fixed upstack!